### PR TITLE
Create new pull request template

### PR DESCRIPTION
**Pull request by thadguidry**
We will eventually want to support different types of Pull requests (issues, docs, etc.) and GitHub supports a Template directory where many templates can be used for different kinds of PR's and can have different automation flows with query parameters.  This will be a useful starting point as we build out more automation (@ostephens docs effort) and expand our contributors.

Details here: 
- https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates
- https://help.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters

### CODE REVIEW

**Comment by wetneb** [comment]
+1 for encouraging to link to the relevant issue. Let's keep this template minimal, we don't want to deter people from submitting PRs either :)

**Comment by thadguidry** [comment]
@wetneb That was the idea... minimal.  For Docs, or in the Docs repo, once we have something or other...then certainly the templates needed there could be different.  For us on OpenRefine trunk, what I have proposed here is all we need for now.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Fixes issue # .
```
Comment (kushthedude) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I think GitHub special keyword is `Fixes #{issue_number}` for automatic closing of issue. 🤔

Comment (thadguidry) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: If you can find it in their docs, that would be great!  See the links I provided above.

Comment (kushthedude) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue
See the linking sub-heading in the above doc!

Comment (thadguidry) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: It's a template, so as long as we have a reminder to them to type in the issue number.  The autosuggest of Issues appears after someone types the number sign # only in fact.   But sure, we can give them a larger hint reminder like you have suggested if you think is needed.

Comment (kushthedude) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: No I was talking about the change `Fixes issue #` to `Fixes #` writing issue ahead of the # will not close the issue.

Comment (thadguidry) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Ah the "supported keywords"!  Yes you are right.

**Pull request merged**